go/interpreter/lfvm/instructions.go

```diff
@@ -1084,19 +1091,20 @@ func opCall(c *context) {
 	// and not doing it would be identified by the replay tool as an error.
 	toAddr := tosca.Address(addr.Bytes20())
 
+	// for static and delegate calls, the following value checks will always be zero.
+
 	// Charge for transferring value to a new address
 	if !value.IsZero() {
 		base_gas += tosca.Gas(params.CallValueTransferGas)
 	}
 
 	// if evm.chainRules.IsEIP158 according to GETH it is EIP158 since 2016
 	// !!!! but need to touch stateDB for the address to have it in the substate record key/value
-	if !value.IsZero() && !c.context.AccountExists(toAddr) {
+	if !value.IsZero() && !c.context.AccountExists(toAddr) && kind == tosca.Call {
```

Comment:
```suggestion
	if kind == tosca.Call && !value.IsZero() && !c.context.AccountExists(toAddr)  {
```

Avoid triggering AccountExists with lazy evaluation easier to read as well. more generic -> more specific